### PR TITLE
Downloads Report: add summary number, chart and table

### LINKED ISSUE #914: Analytics: Downloads Report Chart
**Design Post**: p6riRB-3dU-p2
**Mock API**: [https://app.swaggerhub.com/apis/peterfabian/wc-v3-api/1.0.0#/default/get_reports_downloads_stats](get_reports_downloads_stats)
**REST API Post**: p7bje6-15q-p2

This task involves creating the summary number and chart for the Downloads Report.

Currently the REST API endpoint is not developed yet, so we will need to utilize the Swaggerhub endpoint ( #428 )

### PR DESCRIPTION
**Pull request by Aljullu**
Fixes #914.
Part of #915.

Add Summary Number, Chart and Table to _Downloads_ report.

Under the _Product Title_ and _User Name_ columns, we are displaying the ID for now instead of the title/name (see discussion in #915).

Data is loaded from SwaggerHub.

### Screenshots
image

### Detailed test instructions:
- Go to the _Downloads_ report.
- Verify the summary number, chart and table are displayed and show data (from SwaggerHub for now).

### CODE REVIEW

File: client/analytics/report/downloads/table.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+		return (
+			<ReportTable
+				endpoint="downloads"
+				getHeadersContent={ this.getHeadersContent }
+				getRowsContent={ this.getRowsContent }
+				getSummary={ this.getSummary }
+				query={ query }
+				title={ __( 'Downloads', 'wc-admin' ) }
+			/>
```
Comment (coderkevin) [inline client/analytics/report/downloads/table.js]: Consider adding `columnPrefsKey` for saving user's column visibility preferences?

**Review by coderkevin — APPROVED** [review]
Page views with test data, looks good. Just a couple of comments. Pre-approving.

File: client/analytics/report/downloads/table.js [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+					),
+					value: product_id,
+				},
+				{
+					display: file_name,
+					value: file_name,
+				},
+				{
+					display: (
+						<Link href={ 'post.php?post=' + order_id + '&action=edit' } type="wp-admin">
```
[the remark this answers is not in the document]
Comment (Aljullu) [reply client/analytics/report/downloads/table.js]: Done: 9b4e079fadbbb0e77f18f4d9ddfcff1a5c277cba.

File: client/analytics/report/downloads/table.js (hunk 1 above)
Comment (Aljullu) [reply client/analytics/report/downloads/table.js]: Fixed too: 3f77de81d841981d0a6402087151f1eea5fe99e9.

**Pull request merged**